From 75bdbc4a3ce6e00a29025e0a15455fb92fb9cd7a Mon Sep 17 00:00:00 2001
From: David Edmundson <kde@davidedmundson.co.uk>
Date: Wed, 27 Jan 2021 10:58:20 +0000
Subject: [PATCH 08/10] [systemd] Move kwin to "wants"

As kwin-wayland is still spawned manually by us for Plasma 5.21.


(cherry picked from commit 07b2d67f5b4c8114822c319dc2b8d77fac20a86f)
---
 startkde/systemd/plasma-workspace@.target | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/startkde/systemd/plasma-workspace@.target b/startkde/systemd/plasma-workspace@.target
index ca0bf390d..9fae5fe3c 100644
--- a/startkde/systemd/plasma-workspace@.target
+++ b/startkde/systemd/plasma-workspace@.target
@@ -1,6 +1,6 @@
 [Unit]
 Description=KDE Plasma Workspace
-Requires=plasma-kwin_%i.service plasma-core.target xdg-desktop-autostart.target graphical-session.target
-Wants=plasma-restoresession.service plasma-xembedsniproxy.service plasma-gmenudbusmenuproxy.service plasma-powerdevil.service plasma-ksplash-ready.service plasma-polkit-agent.service plasma-baloo.service plasma-foreground-booster.service
+Requires=plasma-core.target xdg-desktop-autostart.target graphical-session.target
+Wants=plasma-kwin_%i.service plasma-restoresession.service plasma-xembedsniproxy.service plasma-gmenudbusmenuproxy.service plasma-powerdevil.service plasma-ksplash-ready.service plasma-polkit-agent.service plasma-baloo.service plasma-foreground-booster.service
 BindsTo=graphical-session.target
 Before=graphical-session.target xdg-desktop-autostart.target plasma-ksplash-ready.service plasma-restoresession.service
-- 
2.29.2

